### PR TITLE
Rstudio server updates

### PR DESCRIPTION
**Pull request by dtenenba**
There have been significant changes to the way the RStudio Server launch script works, so this PR documents those changes.

We are no longer stuck with a single older version of R. Going forward, users will be able to use RStudio Server with any version of R beginning with fhR/4.1.2-foss-2021b (as of now, for another week or so, that's the only version that will work).

The name of the script has changed too - the old one is kept around for users who want to continue using RStudio Server with `fhR/4.0.2-foss-2019b`.

I have not rolled out these changes yet, want to wait till the docs are out there, but you can see them at

https://github.com/FredHutch/launch-rstudio-server/tree/support-RStudio-Server/1.4.1717

(as of right now there is just one commit in that branch). 

I also think I might be able to see how this could work as a web app, so no more need for this command line script, but that will take a little longer.

### CODE REVIEW

**Comment by dtenenba** [comment]
Whoops, somehow a stray commit from another branch got in there. [This](https://github.com/FredHutch/wiki/pull/706/commits/6cff67c74b61a57bd7e14b065e8382a65531e05e) is the one that has the relevant changes. Will move the others out of the way soon.

File: _scicomputing/software_R.md [hunk 1]
```diff
@@ -84,19 +84,33 @@ To run `R` on a gizmo node, you can follow the same instructions as for `rhino`
...
 
 Here are the steps to run this wrapper.
 
 * Grab a node using the [grabnode](/compdemos/first_rhino/#logging-on-to-gizmo-via-grabnode) command. Specify how many CPUs and how much memory you want, and how many days you want to have control of the node. Remember that you can launch `slurm` jobs within RStudio, so you may not need to ask for a lot of computing power for your RStudio machine.
-* Once you are on the node you grabbed, choose a version of `R` by using the [module load](/scicomputing/compute_environments/#how-to-use-environment-modules) command (`ml` for short). (Example: `fhR/4.0.2-foss-2019b`). **NOTE**: At present, `fhR/4.0.2-foss-2019b` is the *only* R module that will work with RStudio Server.
-* Run the `launch_rstudio_server` command. This will produce a URL that you can paste into your browser. (This URL only works inside the Hutch network, so you need to be on campus or using VPN.)
+* Run the `start_rstudio_server` command. This will produce a URL that you can paste into your browser. (This URL only works inside the Hutch network, so you need to be on campus or using VPN.)
```
Comment (atombaby) [inline _scicomputing/software_R.md:92]: Do we need to change the command name used to start the server?

File: _scicomputing/software_R.md [hunk 2]
```diff
@@ -84,19 +84,33 @@ To run `R` on a gizmo node, you can follow the same instructions as for `rhino`
 There is a wrapper script that allows you to run RStudio Server (the web-browser-based version of RStudio)
 on the HPC machines (cluster machines whose names start with `gizmo`).
 
-You can run this with any version of `R` that is available on our shared computing systems.
+You can run this with any version of `R` that is available on our shared computing systems (beginning with `fhR/4.1.2-foss-2021b`).
```
[the remark this answers is not in the document]
Comment (vortexing) [reply _scicomputing/software_R.md:87]: I agree.  Just write, "You can run this with any version of R that is `fhR/4.1.2-foss-2021b` or later available on our shared computing systems."

**Comment by vortexing** [comment]
@dtenenba just so I am clear, you're wanting to merge this PR into the docs BEFORE you roll out the new setup right?

File: _scicomputing/software_R.md (hunk 1 above)
Comment (vortexing) [reply _scicomputing/software_R.md:92]: I don't mind the change.  Makes it less likely that you'll accidentally do it the "old way" and run into issues.  But any shift will initially confuse people.  Given how much confusion there is though about Rstudio and R versions though, I feel like this extra change that allows them to never think about R versions again hopefully, is worth it.

**Comment by dtenenba** [comment]
This PR is a mess and I am going in a completely different direction, so I am going to close it and make a totally new one.

**Pull request closed without merging**